client/src/common/components/Confirmation.jsx

```diff
@@ -15,15 +16,15 @@ const Confirmation = ({ className, disabled, onCancel, onConfirm, title }) => (
       preloaderTheme={PreloaderTheme.LIGHT}
       type="button"
     >
-      Confirm
+      <FormattedMessage id="common.confirmation.confirm" />
     </PendingButton>
     <button
       className="primary-button"
       disabled={disabled}
       onClick={onCancel}
       type="button"
     >
-      Cancel
+      <FormattedMessage id="common.confirmation.cancel" />
```

Comment:
maybe id name: `common.button.cancel` and `common.button.confirm`?

Reply:
I took the approach
**module . component . string**